deepforest/predict.py

```diff
@@ -18,6 +18,7 @@
 
 def predict_image(model,
                   image,
+                  path,
```

Comment:
Changing the signature of a function is not the best way to go. This causes backward compatibility.

Avoid having the same code in several areas.